frontend/device/sdl/device.lua

```diff
@@ -215,11 +215,21 @@ function Device:init()
         setClipboardText = function(text)
             return input.setClipboardText(text)
         end,
+        gameControllerRumble = function(left_intensity, right_intensity, duration)
+            return input.gameControllerRumble(left_intensity, right_intensity, duration)
+        end,
         file_chooser = input.file_chooser,
     }
 
     self.keyboard_layout = require("device/sdl/keyboard_layout")
 
+    if self.input.gameControllerRumble(0, 0, 0) then
```

Comment:
Note to self: somewhere it should convert -1 into false. Probably best to stick that directly in FFI.